### ISSUE TITLE
Bug: GitHub Connect shows in insights panel when loading the page

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2433]
### Describe the bug

The additional connect with GitHub button appears in the insights panel. This wasn't an issue before we added the ability to show featured insights if you weren't logged in.

https://github.com/open-sauced/app/assets/833231/e668fb4b-f013-4b85-9bab-c6fb90a46f41

### Steps to reproduce

1. log in to the app
2. go to https://app.opensauced.pizza/hub/insights
3. Notice the GitHub connect button in the insights panel.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by priyankarpal** [comment]
yes there is an issue after login if I refresh the page shows the `connect github` button for a second

**Comment by arvindpndit** [comment]
I would like to work on this issue

**Comment by arvindpndit** [comment]
.take
